Emby.Server.Implementations/Localization/LocalizationManager.cs

```diff
@@ -176,82 +170,80 @@ private async Task LoadCultures()
         }
 
         /// <inheritdoc />
-        public IEnumerable<CountryInfo> GetCountries()
+        public IReadOnlyList<CountryInfo> GetCountries()
         {
-            using StreamReader reader = new StreamReader(
-                _assembly.GetManifestResourceStream(CountriesPath) ?? throw new InvalidOperationException($"Invalid resource path: '{CountriesPath}'"));
-            return JsonSerializer.Deserialize<IEnumerable<CountryInfo>>(reader.ReadToEnd(), _jsonOptions)
-                ?? throw new InvalidOperationException($"Resource contains invalid data: '{CountriesPath}'");
+            using var stream = _assembly.GetManifestResourceStream(CountriesPath) ?? throw new InvalidOperationException($"Invalid resource path: '{CountriesPath}'");
```

Comment:
feels like this should rather be cached instead of beeing read each time

Reply:
This is only done once though?

Reply:
nvm then